src/scripts/createClaudeDesktopExtensionManifest.ts

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+import { writeFileSync } from 'fs';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+
+import packageJson from '../../package.json' with { type: 'json' };
+import { ProcessEnvEx } from '../../types/process-env.js';
+import { toolNames } from '../tools/toolName.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+type EnvVar = {
+  type: 'string' | 'number' | 'boolean';
+  title: string;
+  description: string;
+  required: boolean;
+  sensitive: boolean;
+};
+
+type EnvVars = {
+  [TKey in keyof ProcessEnvEx]: EnvVar & { includeInUserConfig: boolean };
+};
+
+const envVars = {
+  SERVER: {
+    includeInUserConfig: true,
+    type: 'string',
+    title: 'Server',
+    description: 'The URL of the Tableau server e.g. https://tableau.example.com',
+    required: true,
+    sensitive: false,
+  },
+  SITE_NAME: {
+    includeInUserConfig: true,
+    type: 'string',
+    title: 'Site Name',
+    description:
+      'The name of the Tableau site to use. For Tableau Server, leave this empty to specify the default site.',
+    required: false,
```

Comment:
SITE_NAME needs to be optional since the user needs to be able to provide an empty value to use the default Tableau site. When required, leaving the field empty caused a blocking error.